### PR TITLE
Fixing Master

### PR DESCRIPTION
**Pull request by biozit**
Fixing Master

### CODE REVIEW

File: etc/cvmfs/domain.d/osgstorage.org.conf [hunk 1]
```diff
@@ -16,40 +16,49 @@ CVMFS_KEYS_DIR=$CVMFS_MOUNT_DIR/$CVMFS_CONFIG_REPOSITORY/etc/cvmfs/keys/osgstora
...
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://dtn-pas.jack.nrp.internet2.edu:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://its-condor-xrootd1.syr.edu:8000/"
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://sc-cache.chtc.wisc.edu:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://stash.farm.particle.cz:8000/"
-# extra OSDF caches not used by OSG
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://fiona-r-uva.vlan7.uvalight.net:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://xcachevirgo.pic.es:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://stashcache.edi.scotgrid.ac.uk:8000/"
+
+
```
Comment (DrDaveD) [inline etc/cvmfs/domain.d/osgstorage.org.conf:61]: Please don't add extra blanks

File: etc/cvmfs/osgstorage-auth.conf [hunk 2]
```diff
@@ -10,42 +10,53 @@ X509_CERT_DIR=
...
+
 # extra authenticated OSDF caches not used by OSG
+# North America
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://sc-cache.chtc.wisc.edu:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://osg-stash-sfu-computecanada-ca.nationalresearchplatform.org:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://osg-hawk-cache.cc.lehigh.edu:8443/"
+# Europe
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://cf-ac-uk-cache.nationalresearchplatform.org:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://ds-914.cr.cnaf.infn.it:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://ligo.hpc.swin.edu.au:8443/"
```
Comment (DrDaveD) [inline etc/cvmfs/osgstorage-auth.conf:51]: Australia should be used by OSG I think.

File: etc/cvmfs/osgstorage-auth.conf [hunk 3]
```diff
@@ -10,42 +10,53 @@ X509_CERT_DIR=
...
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://fiona-r-uva.vlan7.uvalight.net:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://stashcache.jinr.ru:8443/"
```
Comment (DrDaveD) [inline etc/cvmfs/osgstorage-auth.conf:53]: Russia should be considered Asia I think.

**Review by DrDaveD — CHANGES_REQUESTED** [review]
We're getting there.  But shouldn't there be a small number of European caches used by OSG?  And why are there some U.S. caches not used by OSG?

File: etc/cvmfs/osgstorage-auth.conf (hunk 3 above)
Comment (biozit) [reply etc/cvmfs/osgstorage-auth.conf:53]: I am using the location from the GeoIp location.

File: etc/cvmfs/domain.d/osgstorage.org.conf (hunk 1 above)
Comment (biozit) [reply etc/cvmfs/domain.d/osgstorage.org.conf]: fixed

File: etc/cvmfs/osgstorage-auth.conf (hunk 2 above)
Comment (biozit) [reply etc/cvmfs/osgstorage-auth.conf:51]: sure

File: etc/cvmfs/osgstorage-auth.conf (hunk 3 above)
Comment (DrDaveD) [reply etc/cvmfs/osgstorage-auth.conf:53]: JINR is in Russia so if the Geoip data has it somewhere else, the Geoip data is wrong.

**Comment by biozit** [comment]
The City and county are right, it is only the continent

**Comment by DrDaveD** [comment]
> The City and county are right, it is only the continent

Oh then there probably should be a correction to the geoip data, although it doesn't affect the cvmfs geosorting because that only cares about the longitude and latitude.

**Comment by DrDaveD** [comment]
Nevermind. Google says Moscow is actually in Europe.  Apparently Russia is in two continents.

**Comment by biozit** [comment]
I am considering adding all the European caches to OSG. Is there any side effect for it?

**Comment by biozit** [comment]
@DrDaveD Could you check it again?

**Comment by DrDaveD** [comment]
> I am considering adding all the European caches to OSG. Is there any side effect for it?

Yes, the entire list has to be sent to the stratum 1s for looking up every longitude and latitude, and it's not very likely that the osg configuration will be used in Europe so it seems wasteful.  Also potentially it will be a little less maintenance effort in that when new European caches are added they won't necessarily needed to be added to the osg branch.

File: etc/cvmfs/domain.d/osgstorage.org.conf [hunk 4]
```diff
@@ -14,42 +14,45 @@ else
...
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://stash.farm.particle.cz:8000/"
-# extra OSDF caches not used by OSG
+# South America
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://osdf-cache.sprace.org.br:8000/"
+# Asia
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://daejeon-kreonet-net.nationalresearchplatform.org:8000/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://singapore.nationalresearchplatform.org:8000/"
+# Oceania
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://ligo.hpc.swin.edu.au:8000/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://osg-hawk-cache.cc.lehigh.edu:8000/"
```
Comment (DrDaveD) [inline etc/cvmfs/domain.d/osgstorage.org.conf:46]: Isn't Lehigh in Pennsylvania?

File: etc/cvmfs/osgstorage-auth.conf [hunk 5]
```diff
@@ -10,40 +10,49 @@ X509_CERT_DIR=
...
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://osg-hawk-cache.cc.lehigh.edu:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://stashcache.jinr.ru:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://stashcache.ligo.caltech.edu:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://dtn-pas.denv.nrp.internet2.edu:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://dtn-pas.hous.nrp.internet2.edu:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://dtn-pas.jack.nrp.internet2.edu:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://sc-cache.chtc.wisc.edu:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://stash-cache.osg.chtc.io:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://stash.farm.particle.cz:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://ligo.hpc.swin.edu.au:8443/"
```
Comment (DrDaveD) [inline etc/cvmfs/osgstorage-auth.conf:42]: This one should be Oceania I believe.

File: etc/cvmfs/osgstorage-auth.conf [hunk 6]
```diff
@@ -10,40 +10,49 @@ X509_CERT_DIR=
...
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://ds-914.cr.cnaf.infn.it:8443/"
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://stashcache.edi.scotgrid.ac.uk:8443/"
-# extra authenticated OSDF caches not used by OSG
+# Oceania
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://singapore.nationalresearchplatform.org:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://stash-cache.osg.chtc.io:8443/"
```
Comment (DrDaveD) [inline etc/cvmfs/osgstorage-auth.conf:47]: Isn't chtc in Wisconsin?

**Review by DrDaveD — CHANGES_REQUESTED** [review]

File: etc/cvmfs/domain.d/osgstorage.org.conf (hunk 4 above)
Comment (biozit) [reply etc/cvmfs/domain.d/osgstorage.org.conf:46]: fixed

File: etc/cvmfs/osgstorage-auth.conf (hunk 5 above)
Comment (biozit) [reply etc/cvmfs/osgstorage-auth.conf:42]: fixed

File: etc/cvmfs/osgstorage-auth.conf (hunk 6 above)
Comment (biozit) [reply etc/cvmfs/osgstorage-auth.conf]: fixed

**Review by DrDaveD — APPROVED** [review]

**Pull request merged**